static/js/Header.jsx

```diff
@@ -310,6 +310,14 @@ class SearchBar extends Component {
           this.submitSearch(Sefaria.repairCaseVariant(query, d));
           return;
         }
+        if (Sefaria.isGershayimVariant(query, d)) {
+          if (query.includes('"')) {
```

Comment:
Why is this `if` necessary? We already know it's a gershayim variant. Shouldn't we just replace all gershayim with ASCII quotes?

Reply:
how do we know that?

Reply:
Ah! now I understand the issue. Can you wrap this if statement in a function similar to `repairCaseVariant`?